### PR TITLE
Alert on more than 10 errors in 10 minutes, checked every 5 mins

### PR DESCRIPTION
**Pull request by majikandy**
_Alerts to the alerting dashboard based on the defined thresholds and frequency._

_"Relates to issue #409 "_

### Acceptance Criteria

- [ ] ERRORS above the defined frequency create an alert on azure alerts portal https://portal.azure.com/#blade/Microsoft_Azure_Monitoring/AlertsManagementSummaryBlade
- [ ] as above for medicines-api and doc-index-updater

### Testing information

Cause an error  - if you want to affect performance you can use the `gatling` script that is in the scripts folder

### Pre-flight Checklist

- [ ] Testing aligns with [testing strategy][testing strategy]
- [ ] DUXD review approval
- [ ] Accessibility Reviewed
- [ ] Product owner demo

### Pre-merge Checklist

- [ ] Branch test approved

[testing strategy]: https://github.com/MHRA/products/blob/master/docs/principles/testing.md "MHRA/products testing strategy"

### CODE REVIEW

File: medicines/doc-index-updater/scripts/api.scala [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/*
```
Comment (majikandy) [inline medicines/doc-index-updater/scripts/api.scala:1]: without a readme entry on how to use this, it probably doesn't belong in the repo ?

Comment (sgrowe) [reply medicines/doc-index-updater/scripts/api.scala:1]: Yeah, some docs on gatling even if they're brief would be handy

File: infrastructure/modules/cluster/alerts.tf [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  | where ContainerID in (ContainerIdList)
+  | project LogEntrySource, LogEntry, TimeGenerated, Computer, Image, Name, ContainerID
+  | render table
+  | extend message_ = tostring(parse_json(tostring(parse_json(LogEntry).fields)).message)
+  | where parse_json(LogEntry).level == "WARN" or parse_json(LogEntry).level == "ERROR"
+  | where parse_json(tostring(parse_json(LogEntry).fields)).message <> "defaulting SERVICE_BUS_MESSAGE_LOCK_TIMEOUT to \"10\" (environment variable not found)"
+  | extend correlation_id_ = tostring(parse_json(tostring(parse_json(LogEntry).span)).correlation_id)
+  | extend level = tostring(parse_json(tostring(parse_json(LogEntry).level)))
+  | where parse_json(tostring(parse_json(LogEntry).fields)).message <> "defaulting SECONDS_TO_WAIT to \"5\" (environment variable not found)"
+  | where parse_json(tostring(parse_json(LogEntry).fields)).message <> "defaulting SECONDS_TO_WAIT to \"5\" (environment variable not found)"
```
Comment (sgrowe) [inline infrastructure/modules/cluster/alerts.tf:70]: We should probably just set these environment variables rather ignore these logs (also are lines 69 & 70 duplicates?) but I guess that's something we can easily sort later

Comment (majikandy) [reply infrastructure/modules/cluster/alerts.tf:70]: yeah so I think we remove these lines and let it fail - forcing us to address it

File: medicines/doc-index-updater/scripts/api.scala (hunk 1 above)
Comment (majikandy) [reply medicines/doc-index-updater/scripts/api.scala:1]: done

**Pull request merged**